src/test/java/se/sundsvall/datawarehousereader/api/InstallationResourceFailuresTest.java

```diff
@@ -0,0 +1,94 @@
+package se.sundsvall.datawarehousereader.api;
+
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+import static org.springframework.http.MediaType.APPLICATION_PROBLEM_JSON_VALUE;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+import se.sundsvall.datawarehousereader.Application;
+import se.sundsvall.datawarehousereader.service.InstallationService;
+
+@SpringBootTest(classes = Application.class, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("junit")
+class InstallationResourceFailuresTest {
+
+	@MockBean
+	private InstallationService serviceMock;
+
+	@Autowired
+	private WebTestClient webTestClient;
+
+	@Test
+	void getInstallationDetailsPageLessThanMinimum() {
+		webTestClient.get().uri(uriBuilder -> uriBuilder.path("/installations")
+				.queryParam("page", 0)
+				.build())
+			.exchange()
+			.expectStatus().isBadRequest()
+			.expectHeader().contentType(APPLICATION_PROBLEM_JSON_VALUE)
+			.expectBody()
+			.jsonPath("$.title").isEqualTo("Constraint Violation")
```

Comment:
Isn't it better to return an object (from the model class) here instead? I.e:

```
			.expectBody(ConstraintViolationProblem.class)
			.returnResult()
			.getResponseBody();
			
		assertThat(response).isNotNull();
		assertThat(response.getTitle()).isEqualTo("Constraint Violation");
		assertThat(response.getStatus()).isEqualTo(BAD_REQUEST);
		assertThat(response.getViolations()).extracting(Violation::getField, Violation::getMessage).containsExactlyInAnyOrder(
			tuple("page", "must be greater than or equal to 1"));
```

json-path works but are more messy and make refactoring harder (strings instead of methods).

Reply:
I agree,, however this is how it is done in all the other resourcefailuretests in this application. 

We should probably have a standard of what libraries we should be using. 

I feel like mixing is not a good idea, so is it worth the time and effort to refactor the other test classes aswell?